<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/iLogos/logo_novafct.png" width="200">

# Departamento de Engenharia Mecânica e Industrial
## Mecânica Aplicada I 

## EQUILÍBRIO DO CORPO RÍGIDO NO PLANO. ATRITO.

### Problema 6

A figura mostra uma viga AB com massa 200 kg e comprimento de 3 m. A viga está apoiada em A num plano inclinado e apoiada em B num apoio móvel. Existe atrito entre o plano inclinado e a viga $(\mu_\textrm{e}= 0,3)$. Em B existe uma força exterior $\vec{F}$ aplicada de modo a garantir o equilíbrio da viga na posição horizontal.

**a)** Diga o que entende por coeficiente de atrito estático $\mu_\textrm{e}$, por ângulo de atrito estático $\phi_\textrm{e}$ e qual é a relação entre $\mu_\textrm{e}$ e $\phi_\textrm{e}$. Justifique.

**b)** Calcule os valores máximo e mínimo da força $\vec{F}$ para que a viga permaneça em equilíbrio na posição horizontal.


<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch03/P6/MAI_Ch03_P6.png"
width="500">

### Resolução

##### **a)** Diga o que entende por coeficiente de atrito estático $\mu_\textrm{e}$, por ângulo de atrito estático $\phi_\textrm{e}$ e qual é a relação entre $\mu_\textrm{e}$ e $\phi_\textrm{e}$. Justifique.

Esta questão já foi abordada nos problemas propostos 2 e 3. No entanto, a figura abaixo mostra a relação entre o coeficiente de atrito e o ângulo de atrito segundo a lei do atrito seco (ou de Coulomb).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch03/P6/MAI_Ch03_P6a.jpg"
width="500">

#### **b)** Calcule os valores máximo e mínimo da força $\vec{F}$ para que a viga permaneça em equilíbrio na posição horizontal.

**Força mínima**

Para avaliar a força mínima de F (Fmin), para que a viga permaneça em equilíbrio na posição horizontal, é conveniente estudar o pseudo-movimento do mecanismo no intuito de identificar o sentido para a força de atrito no ponto $A$. Para que a viga não “descaia” esta força de atrito deve estar a contrariar esse movimento pelo que, deve ser marcada no DCL para cima. Nestas condições, a força F aplicada no Ponto B tomará um valor mínimo (Fig. 1a). Assim, o DCL da viga deverá desenhar-se conforme se ilustra na Fig. 1(b).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch03/P6/MAI_Ch03_P6b.jpg"
width="700">

Comece-se por escrever as componentes cartesiandas de todas as forças envolvidas no DCL (no cenário de Fmin):

\begin{equation*}
\begin{cases}
\vec{F}_\textrm{A} &=&
\vec{N}_\textrm{A} + \vec{F}_\textrm{aA} \\
&=& N_\textrm{A}\cos(30)\,\vec{\imath}
+ N_\textrm{A}\sin(30)\,\vec{\jmath}
- F_\textrm{aA}\cos(60)\,\vec{\imath}
+ F_\textrm{aA}\sin(60)\,\vec{\jmath} \\
&=&[N_\textrm{A}\cos(30)- F_\textrm{aA}\cos(60)]\,\vec{\imath}
+[N_\textrm{A}\sin(30)+F_\textrm{aA}\sin(60)]\,\vec{\jmath}\\
&=&\left(\displaystyle\frac{\sqrt{3}}{2}N_\textrm{A} -
\frac{1}{2}F_\textrm{aA}\right)\,
\vec{\imath}
+\left(\displaystyle\frac{1}{2}N_\textrm{A} + \frac{\sqrt{3}}{2}F_\textrm{aA}\right)\,
\vec{\jmath}\\
\vec{P} &=& -P\,\vec{\jmath} = mg\,\vec{\jmath} = 200 (9.81)\,\vec{\jmath} \\
\vec{F}_\textrm{min} &=& - F_\textrm{min}\,\vec{\imath}\\
\vec{N}_\textrm{B} &=& N_\textrm{B}\,\vec{\jmath}
\end{cases}
\end{equation*}

A análise do equilíbrio estático do corpo da barra permite escrever (equações  de equilibrio no plano):

\begin{equation*}
\vec{R}=\sum_i\vec{F}_i = 0
\qquad\wedge\qquad
\vec{M}_\textrm{A}^\textrm{R}
= \sum_i\vec{M}_\textrm{A}
= \sum_i(\vec{r}_\textrm{Ai}\times\vec{F}_i) = 0
\end{equation*}

O desenvolvimento das equações algébricas (cartesianas) do equilíbrio estático no plano conduzem ao seguinte sistema linear:

\begin{equation*}
\begin{cases}
\rightarrow(+)~&:~\sum F_\textrm{x} = 0 \\
\uparrow(+)~&:~\sum F_\textrm{y} = 0 \\
\circlearrowleft(+)~&:~\sum M_\textrm{A} = 0 \\
\end{cases}~\Leftrightarrow~
\begin{cases}
\displaystyle\frac{\sqrt{3}}{2}N_\textrm{A} - \frac{1}{2}F_\textrm{aA} -
F_\textrm{min}  &= 0 \\
\displaystyle\frac{1}{2}N_\textrm{A} + \frac{\sqrt{3}}{2}F_\textrm{aA}
+ N_\textrm{B} - P &= 0 \\
- l_{AC}P + l_{AB}N_\textrm{B} &= 0 \\
\end{cases}
\end{equation*}

In [2]:
import numpy as np
import sympy as sy
from sympy.solvers import solve
from sympy.physics.mechanics import ReferenceFrame

mu = 0.3
m = 200 # unit: kg
g = 9.81 # unit: m/s^2

P = m*g
print('P = ','{:.1f}'.format(P),' N')

lAC = 1.5 # unit: m
lCB = lAC
lAB = lAC + lCB
ang60 = np.deg2rad(60) # unit: radianos
ang30 = np.deg2rad(30)

na, faa, fmin, nb = sy.symbols('na faa fmin nb')
S = ReferenceFrame('S')

VFA = (na*np.cos(ang30) - faa*np.cos(ang60))*S.x + \
      (na*np.sin(ang30) + faa*np.sin(ang60))*S.y
vP = -P*S.y
vFmin = -fmin*S.x
vNB = nb*S.y

print('-------------------')
print('Sistema de equações')
print('-------------------')
sumFx = VFA.args[0][0].args[2][0] + vFmin.args[0][0].args[2][0]
print('sumFx = ',sumFx)
sumFy = VFA.args[0][0].args[2][1] + vP.args[0][0].args[2][1] + vNB.args[0][0].args[2][1]
print('sumFy = ',sumFy)
sumMA = -P*lAC + nb*lAB
print('sumMA = ',sumMA)

P =  1962.0  N
-------------------
Sistema de equações
-------------------
sumFx =  -0.5*faa - fmin + 0.866025403784439*na
sumFy =  0.866025403784439*faa + 0.5*na + nb - 1962.0
sumMA =  3.0*nb - 2943.0


Pela lei do atrito: $F_\textrm{aA, max} = \mu_e N_\textrm{A}$, com $\mu_e
= 0.3$. De onde resulta,

\begin{equation*}
\begin{cases}
\sum F_\textrm{x} &=& 0 \\
\sum F_\textrm{y} &=& 0 \\
\sum M_\textrm{A} &=& 0 \\
\end{cases}~\Leftrightarrow~
\begin{cases}
\left[\displaystyle\frac{\sqrt{3}}{2} - \frac{1}{2}(0.3)\right]N_\textrm{A} -
F_\textrm{min}  &=& 0 \\
\left[\displaystyle\frac{1}{2} + \frac{\sqrt{3}}{2}(0.3)\right]N_\textrm{A}
+ N_\textrm{B} - P &=& 0 \\
- l_{AC}P + l_{AB}N_\textrm{B} &=& 0 \\
\end{cases}
\end{equation*}

\begin{equation*}
\qquad\qquad\qquad~\Leftrightarrow~
\begin{cases}
0.716\,N_\textrm{A} -
F_\textrm{min}  &=& 0 \
0.760\,N_\textrm{A}
+ N_\textrm{B} - P &=& 0 \
- 1.5\,P + 1.5\,N_\textrm{B} &=& 0 \
\end{cases}
\end{equation*}

In [14]:
faa = mu*na
print('FaA = ',faa,' N')
VFA = (na*np.cos(ang30) - faa*np.cos(ang60))*S.x + \
      (na*np.sin(ang30) + faa*np.sin(ang60))*S.y

print('-------------------------')
print('Sistema de equações final')
print('-------------------------')
sumFx = VFA.args[0][0].args[2][0] + vFmin.args[0][0].args[2][0]
print('sumFx = ',sumFx)
sumFy = VFA.args[0][0].args[2][1] + vP.args[0][0].args[2][1] + vNB.args[0][0].args[2][1]
print('sumFy = ',sumFy)
print('sumMA = ',sumMA)

print('-------------------')
print('Solução')
print('-------------------')
sol = solve({sumFx,sumFy,sumMA},{na, fmin, nb})
Fmin = sol[fmin]
print(f'Fmin = {Fmin:.1f} N')
NA = sol[na]
print(f'NA   = {NA:.1f} N')
NB = sol[nb]
print(f'NB   = {NB:.1f} N')
print('-------------------')
print('+ F atrito')
print('-------------------')
FAA = mu*NA
print('FaA  = ','{:.2f}'.format(FAA),' N')

FaA =  0.3*na  N
-------------------------
Sistema de equações final
-------------------------
sumFx =  -fmin + 0.716025403784439*na
sumFy =  0.759807621135332*na + nb - 1962.0
sumMA =  3.0*nb - 2943.0
-------------------
Solução
-------------------
Fmin = 924.5 N
NA   = 1291.1 N
NB   = 981.0 N
-------------------
+ F atrito
-------------------
FaA  =  387.33  N


Em altenativa, na forma matricial,

\begin{equation*}
[A]\{x\}=\{b\} \Leftrightarrow
\begin{bmatrix}
-1 & 0.716 & 0  \\
0 & 0.760 & 1 \\
0 & 0 & 1.5 \\
\end{bmatrix}
\begin{bmatrix}
F_\textrm{min} \\ N_\textrm{A} \\ N_\textrm{B}
\end{bmatrix}=
\begin{bmatrix}
0 \\ P \\ 1.5 P
\end{bmatrix}
\end{equation*}

cuja solução é: $\{x\}=[A]^{-1}\{b\}$

In [19]:
import numpy as np
np. set_printoptions(precision=1, suppress=True)

A0 = [-1, np.sqrt(3)/2-1/2*0.3, 0]
A1 = [0, 1/2+np.sqrt(3)/2*0.3, 1]
A2 = [0, 0, lAB]

A = np.array([A0,A1,A2])
print(f'A =\n {A}')

invA = np.linalg.inv(A)
print(f'A^-1 =\n {invA}')

b = np.array([0, P, lAC*P])
print(f'b = {b}')
print('-------------------')
print('Solução')
print('-------------------')
x = np.linalg.solve(A,b)
print(f'x = {x} N')
print('+ F atrito: -------')
print('FaA = ','{:.3f}'.format(x[1]*mu),' N')

A =
 [[-1.   0.7  0. ]
 [ 0.   0.8  1. ]
 [ 0.   0.   3. ]]
A^-1 =
 [[-1.   0.9 -0.3]
 [ 0.   1.3 -0.4]
 [ 0.   0.   0.3]]
b = [   0. 1962. 2943.]
-------------------
Solução
-------------------
x = [ 924.5 1291.1  981. ] N
+ F atrito: -------
FaA =  387.335  N


**Força máxima**

Para calcular o valor da força máxima de F (Fmax) é útil definir o sentido da força de atrito em $A$. Nesta situação, para que a viga não _suba_, a força de atrito deve estar a contrariar esse movimento pelo que, deve-se marcar o sentido da força de atrito no Ponto A com sentido para baixo. Deste modo, a força F aplicada ao Ponto B tomará um valor máximo (Fig.  2a) e o DCL da viga fica conforme se ilustra na figura (Fig 2b).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch03/P6/MAI_Ch03_P6c.jpg"
width="500">

As equações escalares de equilíbrio estático no plano vêm:

\begin{equation*}
\begin{cases}
\sum F_\textrm{x} = 0 \\
\sum F_\textrm{y} = 0 \\
\sum M_\textrm{A} = 0 \\
\end{cases}
\Leftrightarrow
\begin{cases}
\frac{\sqrt{3}}{2}N_\textrm{A} + \frac{1}{2}F_\textrm{aA} -
F_\textrm{max} = 0 \\
\frac{1}{2}N_\textrm{A} - \frac{\sqrt{3}}{2}F_\textrm{aA}
+ N_\textrm{B} - P = 0 \\
- l_{AC}P + l_{AB}N_\textrm{B} = 0 \\
\end{cases}
\end{equation*}

com $F_\textrm{AA, max} = \mu_e N_\textrm{A} = 0.3 N_\textrm{A}$,

\begin{equation*}
\begin{cases}
\sum F_\textrm{x} = 0 \\
\sum F_\textrm{y} = 0 \\
\sum M_\textrm{A} = 0 \\
\end{cases}
\Leftrightarrow
\begin{cases}
\left[\frac{\sqrt{3}}{2} + \frac{1}{2}(0.3)\right] N_\textrm{A} - F_\textrm{max}  = 0 \\
\left[\frac{1}{2} - \frac{\sqrt{3}}{2}(0.3)\right]N_\textrm{A} + N_\textrm{B} - P = 0 \\
- l_{AC}P + l_{AB}N_\textrm{B} = 0 \\
\end{cases}
\end{equation*}

In [5]:
fmax = sy.symbols('fmax')

VFA = (na*np.cos(ang30) + faa*np.cos(ang60))*S.x + \
      (na*np.sin(ang30) - faa*np.sin(ang60))*S.y
vFmax = -fmax*S.x

sumFx = VFA.args[0][0].args[2][0] + vFmax.args[0][0].args[2][0]
print('sumFx = ',sumFx)
sumFy = VFA.args[0][0].args[2][1] + vP.args[0][0].args[2][1] + \
        vNB.args[0][0].args[2][1]
print('sumFy = ',sumFy)
print('sumMA = ',sumMA)

sol = solve({sumFx,sumFy,sumMA},{na, fmax, nb})
Fmax = sol[fmax]
print('Fmax = ','{:.2f}'.format(Fmax),' N')
NA = sol[na]
print('NA = ','{:.2f}'.format(NA),' N')
FAA = mu*NA
print('FAA = ','{:.2f}'.format(FAA),' N')
NB = sol[nb]
print('NB = ','{:.2f}'.format(NB),' N')

sumFx =  -fmax + 1.01602540378444*na
sumFy =  0.240192378864668*na + nb - 1962.0
sumMA =  3.0*nb - 2943.0
Fmax =  4149.68  N
NA =  4084.23  N
FAA =  1225.27  N
NB =  981.00  N


Em resumo, distinguem-se as duas situações anteriormente analisadas, i.e. a situação de força mínima (Fmin) e a situação de força máxima (Fmax) conforme se ilustra pelas Fig. 3 (a) e (b), respectivamente. A conclusão a que se chega é que, para um intervalo de forças entre 924 N e 4150 N a viga AB permanecerá
imóvel e sempre na posição horizontal.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch03/P6/MAI_Ch03_P6d.jpg"
width="700">

---

Copyright (c) DEMI - FCT NOVA

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>